src/views/Home/components/LostAndFoundCard/index.tsx

```diff
@@ -76,7 +90,7 @@ const LostAndFoundCard = () => {
   }, []);
 
   // The row labelling the column names for the report grid
-  const reportHeader = () => (
+  const reportHeader = (title: string) => (
```

Comment:
Is this new title argument used anywhere?

Reply:
I initially used it for the Recently Found, and didnt remove that after i changed the approach

Reply:
Actually its Used, just realised
{reportHeader('Recently Found')}
            {foundReports.map((report) => reportRow(report, true))}

Reply:
Yes, but the title is never used inside the reportHeader component.  If it's not being used inside of it, we should remove it from the component and where it is called.

<img width="647" alt="Screen Shot 2024-12-05 at 14 34 06" src="https://github.com/user-attachments/assets/c192a8c0-9629-46ff-a07b-5d52e052bc5f">